actions/DisplayAction.php

```diff
@@ -43,7 +43,7 @@ public function execute(array $request)
         $noproxy = array_key_exists('_noproxy', $request)
             && filter_var($request['_noproxy'], FILTER_VALIDATE_BOOLEAN);
 
-        if (defined('PROXY_URL') && PROXY_BYBRIDGE && $noproxy) {
+        if (!empty(Configuration::getConfig('proxy', 'url')) && Configuration::getConfig('proxy', 'by_bridge') && $noproxy) {
```

Comment:
I think we can safely replace emptycheck with truthy check:
```php
if (!empty(Configuration::getConfig('proxy', 'url'))
```
=>
```php
if(Configuration::getConfig('proxy', 'url'))
```

Reply:
Makes sense to me.